src/map/mob.c

```diff
@@ -4024,16 +4307,26 @@ static void mob_read_db_drops_sub(struct mob_db *entry, struct config_setting_t
 			i++;
 			continue;
 		}
-		if (mob->get_const(drop, &i32) && i32 >= 0) {
-			value = i32;
+
+		struct optdrop_group *drop_option = NULL;
```

Comment:
move it to the top of the loop, where other variables are declared.

Reply:
hmm, I am a bit confused now, on an earlier review it was requested to move the declaration to where it is used, and now the opposite?

Reply:
old way was correct. now we not using outdated visual studio, and code should follow good style

Reply:
separate declaration is useless and error prone. as many as possible vars must be used and smaller scope